README.md

```diff
@@ -163,103 +174,109 @@ Inspired by [ziadoz/awesome-php](https://github.com/ziadoz/awesome-php)
 * [Laravel Langman](https://github.com/themsaid/laravel-langman) - Manage language files from Artisan Console
 * [Laravel Translation](https://github.com/waavi/translation) - Allow live edit/caching of translation entries, and localization of urls and Eloquent Model attributes.
 
 ##### Third-party Service Integration
-* [Laravel Algolia](https://github.com/vinkla/algolia) - Algolia API bridge
+
+* [Laravel Algolia](https://github.com/vinkla/laravel-algolia) - Algolia API bridge
 * [Laravel Analytics](https://github.com/spatie/laravel-analytics) - An opinionated Laravel 5 package to retrieve pageviews and other data from Google Analytics
 * [Laravel DigitalOcean](https://github.com/GrahamCampbell/Laravel-DigitalOcean) - DigitalOceanV2 bridge
 * [Laravel Dropbox](https://github.com/GrahamCampbell/Laravel-Dropbox) - Dropbox bridge
-* [Laravel Facebook](https://github.com/schimpanz/Laravel-Facebook) - Facebook API bridge
+* [Laravel Facebook](https://github.com/vinkla/laravel-facebook) - Facebook API bridge
 * [Laravel GitHub](https://github.com/GrahamCampbell/Laravel-GitHub) - PHP GitHub API bridge
-* [Laravel GitLab](https://github.com/vinkla/gitlab) - GitLab API bridge
+* [Laravel GitLab](https://github.com/vinkla/laravel-gitlab) - GitLab API bridge
 * [Laravel Googletagmanager](https://github.com/spatie/laravel-googletagmanager) - Easily setup and send data to Google Tag Manager
-* [Laravel Instagram](https://github.com/vinkla/instagram) - Instagram API bridge
+* [Laravel Instagram](https://github.com/vinkla/laravel-instagram) - Instagram API bridge
 * [Laravel Newsletter](https://github.com/spatie/laravel-newsletter) - Send newsletters with Mailchimp
 * [Laravel Parse](https://github.com/GrahamCampbell/Laravel-Parse) - PHP Parse SDK bridge
-* [Laravel Pusher](https://github.com/vinkla/pusher) - Pusher API bridge
-* [Laravel Pushwoosh](https://github.com/schimpanz/Laravel-Pushwoosh) - Pushwoosh API bridge
-* [Laravel Vimeo](https://github.com/vinkla/vimeo) - Vimeo API bridge
+* [Laravel Pusher](https://github.com/vinkla/laravel-pusher) - Pusher API bridge
+* [Laravel Pushwoosh](https://github.com/hoymultimedia/Laravel-Pushwoosh) - Pushwoosh API bridge
+* [Laravel Vimeo](https://github.com/vinkla/laravel-vimeo) - Vimeo API bridge
 
 
 ## Development Setup
-* [Homestead](http://laravel.com/docs/homestead) - Official Vagrant box for Laravel
+
+* [Homestead](https://laravel.com/docs/5.3/homestead) - Official Vagrant box for Laravel
   * [Getting Started with Laravel Homestead](https://scotch.io/tutorials/getting-started-with-laravel-homestead)
   * [Installation on OSx and Linux](https://laracasts.com/series/laravel-5-from-scratch/episodes/3)
   * [Installation on  Windows](http://blog.teamtreehouse.com/laravel-homestead-on-windows)
-* [Valet](http://laravel.com/docs/valet) - development environment for Mac minimalists
+* [Valet](https://laravel.com/docs/5.3/valet/) - development environment for Mac minimalists
```

Comment:
Maybe, the tests will fail though. Lets keep it like this for now and update it in the future. We're only required to update the version number once or twice a year.